### ISSUE TITLE
2.2.0.Beta.3 no longer outputs media queries via susy-breakpoint

### ISSUE THREAD

**Issue by trafiq** [ericam/susy#410]
Hello, I've been testing the libsass branch and all has been working well until the last update. I noticed that commit b82d31587abf2dbe1e215c0321d7ee70566b640b makes some changes to `susy-breakpoint` so suspect that's the culprit. I initially thought it may have been the result of a syntax change but not as far as I can tell from the commit.

For the record, the problem occurs when using 2.2.0.beta.3 with both Ruby Sass and Libsass. I do also have the latest Breakpoint 2.5.0 installed.

Any ideas?

Thanks.

**Comment by trafiq** [comment]
Thanks. I can confirm that the problem has been fixed in Susy 2.2.0.rc.2. So closing the issue.

**Comment by vongoh** [comment]
I am now having this same issue using this setup:

- grunt-sass ^0.17.0
- susy 2.2.2
- breakpoint-sass 2.5.0

For example: 
~~~
// SCSS
.l-content {
    @include span(4);

    @include breakpoint(768px) {
        @include span(3 of 4);
    }
}
// CSS output
.l-content {
    width: 100%;
    float: left;
    margin-left: 0;
    margin-right: 0;
}

@media and (min-width: 768px) {
    .l-content {
        width: 73.6842105263%;
        float: left;
        margin-right: 5.2631578947%;
    }
}
~~~

There's that extra `and` again in the output, its killing the media queries.   I'm a bit out of my depth here, but I looked at the fix in 4d92468 and compared it to what is in the `_breakpoint-plugin.scss` in my 2.2.2 install  - and the code does not match:

~~~
// What I am seeing in your 4d92468 fix .. 
 $no-query: false
) { 
@if susy-support(breakpoint, (mixin: breakpoint), $warn: false) {
     @include breakpoint($query, $no-query) {
         @content;
     }
} @else {
    @media #{susy-build-breakpoint($query)} {
        @content;

// What is in 2.2.2, relevant bits .. 
  $no-query: $susy-media-fallback
) {
  @if susy-support(breakpoint, (mixin: breakpoint), $warn: false) {
    @include breakpoint($query, $no-query) {
      @content;
    }
  } @else {
    @include susy-media($query, $no-query) {
      @content;
~~~

Its different, currently I do not have the shkeelz to analyse this.  Is there any other reason this might be happening?  Is there are version chain ala libass/susy/breakpoint that is currently working?  thanks man!
